tests/test_uri.py

```diff
@@ -118,9 +118,11 @@ def test_pandora_repr_converts_to_string():
     assert obj.uri == uri + "0"
 
 
-def test_pandora_parse_none_mock_uri():
-    uri = PandoraUri()
-    assert uri.encode(None) == ""
+# TODO: I don't think this test is needed anymore... no need
+# to encode strings in python3, right?
+# def test_pandora_parse_none_mock_uri():
```

Comment:
Maybe just skip the test for now by adding the `@pytest.mark.skip` decorator.

Reply:
I'm happy to add the decorator.